core/src/main/java/com/yahoo/ycsb/Status.java

```diff
@@ -87,6 +87,6 @@ public boolean equals(Object obj) {
   public static final Status BAD_REQUEST = new Status("BAD_REQUEST", "The request was not valid.");
   public static final Status FORBIDDEN = new Status("FORBIDDEN", "The operation is forbidden.");
   public static final Status SERVICE_UNAVAILABLE = new Status("SERVICE_UNAVAILABLE", "Dependant service for the current binding is not available.");
-  
+  public static final Status BATCHED_OK = new Status("BATCHED_OK", "The operation has been batched by the binding to be executed later.");
```

Comment:
if you're going to expose this here, please keep it in an isolated commit that also moves the mongo uses of BATCHED_OK to use the newly generalized status result.